hack/cherry_pick_pull.sh

```diff
@@ -16,22 +16,24 @@
 
 # Checkout a PR from GitHub. (Yes, this is sitting in a Git tree. How
 # meta.) Assumes you care about pulls from remote "upstream" and
-# checks thems out to a branch named:
+# checks them out to a branch named:
 #  automated-cherry-pick-of-<pr>-<target branch>-<timestamp>
 
 set -o errexit
 set -o nounset
 set -o pipefail
 
-declare -r KUBE_ROOT="$(dirname "${BASH_SOURCE}")/.."
-cd "${KUBE_ROOT}"
+declare -r REPO_ROOT="$(git rev-parse --show-toplevel)"
```

Comment:
```
In hack/cherry_pick_pull.sh line 26:
declare -r REPO_ROOT="$(git rev-parse --show-toplevel)"
           ^-- SC2155: Declare and assign separately to avoid masking return values.
```

Reply:
fixed